public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -925,6 +933,23 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
       }
     });
   }
+
+  public onSetScrollRef = (scrollElement: ScrollRefElement): void => {
+    this._scrollRef = scrollElement;
+  };
+
+  public rememberScrollPos() {
+    this._prevScrollPos = this._scrollRef?.scrollTop;
+    console.log('rememberScrollPos', this._prevScrollPos);
+  }
+
+  public restoreScrollPos() {
+    if (this._prevScrollPos !== undefined) {
+      setTimeout(() => {
+        this._scrollRef?.scrollTo(0, this._prevScrollPos!);
+      }, 50);
```

Comment:
this setTimeout is annoying but without it does not work, tried requestAnimationFrame but did work either.